src/pages/project/project_wizard/RelatedProjectsField.tsx

```diff
@@ -0,0 +1,76 @@
+import { Autocomplete, Box } from '@mui/material';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFetchProjects } from '../../../api/hooks/useFetchProjects';
+import { AutocompleteTextField } from '../../../components/AutocompleteTextField';
+import { CristinProject, ProjectFieldName, SaveCristinProject } from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+import { RelatedProjectItem } from './RelatedProjectItem';
+
+export const RelatedProjectsField = () => {
+  const { t } = useTranslation();
+  const { values, setFieldValue } = useFormikContext<SaveCristinProject>();
+  const [searchTerm, setSearchTerm] = useState('');
+  const debouncedSearchTerm = useDebounce(searchTerm);
+  const projectsQuery = useFetchProjects(debouncedSearchTerm);
+  const projects = projectsQuery.data?.hits ?? [];
+
+  const removeProject = (projectId: string) =>
+    setFieldValue(
+      ProjectFieldName.RelatedProjects,
+      values[ProjectFieldName.RelatedProjects].filter((val) => val !== projectId)
+    );
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+      <Field name={ProjectFieldName.RelatedProjects}>
+        {({ field, form: { setFieldValue } }: FieldProps<string[]>) => (
+          <Autocomplete
+            data-testid={dataTestId.registrationWizard.description.projectForm.relatedProjectsSearchField}
+            options={projects}
+            getOptionLabel={(option) => option.title || ''}
+            onInputChange={(_, newInputValue, reason) => {
+              if (reason !== 'reset') {
+                // Autocomplete triggers "reset" events after input change when it's controlled. Ignore these.
+                setSearchTerm(newInputValue);
+              }
+            }}
+            inputValue={searchTerm}
+            onChange={(_, value: CristinProject | null) => {
+              if (value?.id) {
+                setSearchTerm('');
+                const projectUris = [...values[ProjectFieldName.RelatedProjects], value?.id];
```

Comment:
`values` er typed her, så kan ev bruke den direkte også. Jeg syns det gir litt bedre readability, men tror dette også vil gi feilmelding om enumen gir en streng som ikke er definert i typen, så blir jo det samme uansett. Så greit om du liker å bruke enumen bedre også!
```suggestion
                const projectUris = [...values.relatedProejcts, value?.id];
```